src/tests/multihost/ad/test_sudo.py

```diff
@@ -71,18 +78,19 @@ def test_001_bz1380436(self, multihost):
                         assert '/usr/bin/less\n' in result
 
     def test_002_bz1372440(self, multihost):
-        """@Title: IDM-SSSD-TC: ad_provider: ad_sudo: AD managed sudo groups
-        will not work with sssd
-
-        @setup:
-        1. Add sudo rules with sudoUser attribute set to group names
-           (%sudo_idmgroup1 which has member sudo_idmuser1)
-        2. Add users to the group.
-        3. List the sudo commands allowed for the user
-
-        @expectedResuls:
-        1. Verify sudo_idmuser1 can fetch the sudo rule and run
-        the required command  as sudo
+        """
+        :title: IDM-SSSD-TC: ad_provider: ad_sudo: AD managed sudo groups
+         will not work with sssd
+        :id: 56616411-d56a-4e3f-b732-a39a4fae8bbc
+        :setup:
+          1. Add sudo rules with sudoUser attribute set to group names
+             (%sudo_idmgroup1 which has member sudo_idmuser1)
+          2. Add users to the group.
+          3. List the sudo commands allowed for the user
+
+        :expectedresults:
```

Comment:
here we can rename setup as steps and expectedresults should contain all expected results for all points mentioned on  steps .

:steps:
1.
2.
:expectedresults:
1. 
2.

Reply:
done